db/migrate/20181016140921_migrate_orch_stacks_to_have_ownership_concept.rb

```diff
@@ -37,8 +37,7 @@ class MiqGroup < ActiveRecord::Base
 
     def self.create_tenant_group(tenant)
       create_with(
-        :description         => "Tenant access",
-        :default_tenant_role => MiqUserRole.default_tenant_role
```

Comment:
default_tenant_role was never an attribute on MiqGroup.  🤷‍♂️  (rails 5.2 raises unknown attribute error if you try to update with unknown attributes)